Installation/Action/kiss.EmailContactmomentToAfdelingAction.action.json

```diff
@@ -0,0 +1,53 @@
+        "variables": {
+            "klantVoornaam": "response.embedded.partij.embedded.persoon.voornaam",
+            "klantTussenvoegsel": "response.embedded.partij.embedded.persoon.voorvoegselAchternaam",
+            "klantAchternaam": "response.embedded.partij.embedded.persoon.achternaam",
+            "specificatie": "response.embedded.klantcontact.onderwerp",
+            "notitie": "response.embedded.klantcontact.todo",
+            "afdeling": "response.embedded.klantcontact.embedded.actors.0.embedded.organisatorischeEenheid.eMailadres",
+            "medewerker": "response.embedded.klantcontact.embedded.actors.1.embedded.medewerker.eMailadres"

```

Comment:
This line can be removed

Reply:
The medewerker who created the contactverzoek is mentioned in the template